src/main/java/org/sunbird/workflow/service/UserBulkUploadService.java

```diff
@@ -259,8 +260,8 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                                     userProfileWfService.updateUserProfile(wfRequest);
                                     WfStatusEntity wfStatusEntityFailed = wfStatusRepo.findByWfId(wfRequest.getWfId());
                                     if(Constants.REJECTED.equalsIgnoreCase(wfStatusEntityFailed.getCurrentStatus())){
```

Comment:
if the existing workflow is in rejected status -- the code is also removing that particular field from the update action -- this will prevent updating of the particular fields forever by the user / admin ... 
which is not mentioned anywhere in the user story... 
you should fetch only the pending approval requests from workflow table and process it -- if you are doing so, then this code is unnecessary...

Reply:
This Rejected status is being checked if there is if the update fails in learner service, in the normal flow the same request will go ahead for the update ignoring this Rejected request. This rejected record in redundant for the flow I have written its only used to check if the update has failed.